src/app/collaborators/page.tsx

```
@@ -79,144 +142,333 @@ export default function CollaboratorsPage() {
+                <svg xmlns="http://www.w3.org/2000/svg" className="h-5 w-5" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                  <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M5 15l7-7 7 7" />
+                </svg>
+              </a>
+            </div>
           </Container>
-        </main>
+        </div>
       </div>
+      
+      {/* Add a custom scroll progress indicator */}
+      <div className="fixed bottom-0 left-0 h-1 bg-[color:var(--color-accent)]" style={{ width: 'var(--scroll-width, 0%)' }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)